server.js

```diff
@@ -277,22 +277,27 @@ app
                     .toString()
                     .padStart(2, "0") +
                   "-" +
-                  date
-                    .getUTCDate()
-                    .toLocaleString(undefined, { minimumIntegerDigits: 2 }) +
+                  date.getUTCDate().toLocaleString(undefined, {
+                    minimumIntegerDigits: 2,
+                  }) +
                   " " +
                   date.getUTCHours() +
                   ":" +
-                  date
-                    .getUTCMinutes()
-                    .toLocaleString(undefined, { minimumIntegerDigits: 2 }) +
+                  date.getUTCMinutes().toLocaleString(undefined, {
+                    minimumIntegerDigits: 2,
+                  }) +
                   " (UTC)",
-                upload_progress: job.progress(),
+                upload_progress: `${job.progress().step}:${
+                  job.progress().value
+                }`,
```

Comment:
This is not backward compatible

![Screenshot 2023-12-12 at 22-41-34 Book Uploader Bot](https://github.com/coderwassananmol/BUB2/assets/57034486/7a7caa9c-e4ce-430f-881c-d9015bfe4b36)

Older uploads return progress statuses of undefined since they do not have a `step` or `value` values. Figure out how to differentiate and return appropriate progress updates.

Reply:
@Parthiv-M I have made the requested changes